ui.html

```diff
@@ -0,0 +1,259 @@
+				height: 100%;
+			}
+			.heading {
+				color: #ececf9;
+				font-weight: bold;
+				margin-bottom: 28px;
+			}
+			.white-text {
+				color: #ececf9 !important;
+			}
+			.checkbox__label::before {
+				border: 1px solid white !important;
+			}
+			.description {
+				margin-bottom: 4px;
+				margin-top: 4px;
+				color: #aaabc5 !important;
+				font-size: 10px;
+				padding-left: 2px;
+			}
+			.reset-container {
+				display: flex;
+				gap: 4px;
+				align-items: center;
+				cursor: pointer;
+			}
+			.reset-text {
+				color: #18a0fb;
+				font-size: 12px;
+			}
+			.reset-icon {
+				height: 12px !important;
+				width: 12px !important;
+			}
+			.checkbox__box {
+				cursor: pointer;
+			}
+		</style>
+
+		<script
+			crossorigin="anonymous"
+			src="https://cdn.jsdelivr.net/gh/thomas-lowry/figma-plugin-ds/dist/iife/figma-plugin-ds.js"
+		></script>
+	</head>
+
+	<body class="main-container">
+		<link
+			rel="stylesheet"
+			href="https://cdn.jsdelivr.net/npm/figma-plugin-ds@1.0.1/dist/figma-plugin-ds.min.css"
+		/>
+		<link rel="stylesheet" href="style.css" />
+		<div>
+			<div class="heading">Erase Bg</div>
+			<!-- <div class="icon icon--spinner icon--spin"></div> -->
+			<div id="formContainer"></div>
+		</div>
+
+		<div class="bottom-btn-container">
+			<div class="reset-container" id="reset">
+				<div class="icon icon--swap icon--blue reset-icon"></div>
+				<div class="reset-text">Reset all</div>
+			</div>
+
+			<button id="apply" class="button button--primary">Apply</button>
+		</div>
+
+		<!-- --------------------- SCRIPT START ------------------------- -->
+		<script crossorigin="anonymous">
+			let bytes = null,
+				optionsArray = [];
+			prevSavedFormValues = "";
+
+			parent.postMessage(
+				{
+					pluginMessage: {
+						type: "initialCall",
+					},
+				},
+				"*"
+			);
+
+			function camelCase(str) {
+				return str
+					.replace(/(?:^\w|[A-Z]|\b\w)/g, function (word, index) {
+						return index == 0 ? word.toLowerCase() : word.toUpperCase();
+					})
+					.replace(/\s+/g, "");
+			}
+
+			window.onmessage = async (event) => {
+				const { data } = event;
+				if (data.pluginMessage.type === "createForm") {
+					optionsArray = data.pluginMessage.optionsArray;
+					prevSavedFormValues = data.pluginMessage.savedFormValue;
+					createOptionsForm(false);
+				}
+				if (data.pluginMessage.type === "selectedImage") {
+					bytes = data.pluginMessage.imageBytes;
+					// console.log("bytes", data.pluginMessage.imageBytes);
+					const pixelbin = new Pixelbin({
+						cloudName: "muddy-lab-41820d",
+						zone: "default",
+					});
+					const image = await pixelbin.image(
+						"__playground/playground-default.jpeg"
+					);
+					let t = Pixelbin.transformations.EraseBG.bg();
+					image.setTransformation(t);
+					// console.log("url after remoing bg", image.getUrl());
+				}
+			};
+
+			function createOptionsForm(isReset) {
+				var formHolder = document.getElementById("formContainer");
```

Comment:
rename formContainer to something more relevant

Reply:
changes done